contract-tests/sse-contract-tests/src/entity_manager.cpp

```diff
@@ -35,6 +35,11 @@ std::optional<std::string> EntityManager::create(ConfigParams const& params) {
             std::chrono::milliseconds(*params.readTimeoutMs));
     }
 
```

Comment:
This was missing, but it's already implemented. Easy win for speeding up contract tests.